Rakefile

```diff
@@ -1,7 +1,54 @@
+require 'pg'
```

Comment:
Just to confirm what I think we found out yesterday morning - this all looks fine.  Other people should be able to use your Rake tasks to set up their DB environment and run your app.